src/components/Loader.vue

```diff
@@ -395,4 +395,23 @@ export default Loader;
         max-width: 70%;
         opacity: 1;
     }
+
+    .loader.exit-transition {
+        /* animation: exit-transition 600ms 1s; */
+    }
+
+    @keyframes exit-transition {
+        from { transform: scale(1); opacity: 1; }
+        80%  { opacity: 0; }
+        to   { transform: scale(0); opacity: 0; }
+    }
+
+    .loader.exit-transition .success .icon-row {
```

Comment:
What's actually the reason for the unscoped second `<style>` tag and manual scoping instead?

Reply:
Slots do not inherit scoped styles. Thus when we use custom HTML for slots from outside, those can only be styled with global styles.